### PR TITLE
JEP-236 Add timeline for Java 8 EOL

### PR DESCRIPTION
**Pull request by timja**
Dates have been agreed after discussions with Mark and security team

### CODE REVIEW

File: jep/236/README.adoc [hunk 1]
```diff
@@ -229,15 +229,14 @@ Review link:https://issues.jenkins.io/issues/?jql=resolution%20%3D%20Unresolved%
...
 
-==== Timeline (TBD)
-
-* Announce Java 8 end of support for weekly in blogpost
-** Describe Java 11 upgrade process for users of war, deb, rpm, and msi installations
-* Add higher visibility warning on Java 8 end of support in Jenkins weekly (an idea that is not yet fully formed)
-* Remove Java 8 from weekly core release, weekly Docker controller images
-* Announce Java 8 end of support for LTS in blogpost
-* Remove Java 8 from LTS core release, include in changelog and upgrade guide
-* Remove Java 8 from Docker agent images
```
Comment (timja) [inline jep/236/README.adoc:240]: I don't see a need to mention these here, they should just track weekly and LTS removals when maintainers get to them

Comment (MarkEWaite) [reply jep/236/README.adoc:240]: Agreed

**Comment by MarkEWaite** [comment]
The changes look good to me.

File: jep/236/README.adoc [hunk 2]
```diff
@@ -229,15 +229,14 @@ Review link:https://issues.jenkins.io/issues/?jql=resolution%20%3D%20Unresolved%
...
+==== Timeline
+
+* 2022-05 Announce Java 8 end of support for weekly in blogpost
+** Describe Java 11 upgrade process for users of war, deb, rpm, and msi installations - link:https://www.jenkins.io/doc/administration/requirements/upgrade-java-guidelines/#upgrading-jenkins[Completed]
+* 2022-05-17 Add higher visibility warning on Java 8 end of support in Jenkins weekly
+* 2022-05-31 Remove Java 8 from weekly core release, weekly Docker controller images
```
Comment (timja) [inline jep/236/README.adoc:237]:
```suggestion
* 2022-06-21 Remove Java 8 from weekly core release, weekly Docker controller images
```

**Review by MarkEWaite — APPROVED** [review]
I think this is the right plan.  Thanks!

**Review by daniel-beck — DISMISSED** [review]
As above. If you wanted this to happen before September 1 in weeklies, you should have written the admin monitor to state that.

File: jep/236/README.adoc [hunk 3]
```diff
@@ -229,15 +229,14 @@ Review link:https://issues.jenkins.io/issues/?jql=resolution%20%3D%20Unresolved%
...
-* Remove Java 8 from weekly core release, weekly Docker controller images
-* Announce Java 8 end of support for LTS in blogpost
-* Remove Java 8 from LTS core release, include in changelog and upgrade guide
-* Remove Java 8 from Docker agent images
+==== Timeline
+
+* 2022-05 Announce Java 8 end of support for weekly in blogpost
+** Describe Java 11 upgrade process for users of war, deb, rpm, and msi installations - link:https://www.jenkins.io/doc/administration/requirements/upgrade-java-guidelines/#upgrading-jenkins[Completed]
+* 2022-05-17 Add higher visibility warning on Java 8 end of support in Jenkins weekly
+* 2022-06-21 Remove Java 8 from weekly core release, weekly Docker controller images
```
[the remark this answers is not in the document]
Comment (MarkEWaite) [reply jep/236/README.adoc:237]: The proposal is to change that warning in weekly releases to state that the support of Java 8 will end after June 20, 2022.

**Comment by MarkEWaite** [comment]
> As above. If you wanted this to happen before September 1 in weeklies, you should have written the admin monitor to state that.

We intend to update the admin monitor to state that.

File: jep/236/README.adoc (hunk 3 above)
[the remark this answers is not in the document]
Comment (daniel-beck) [reply jep/236/README.adoc:237]: Doesn't change anything about the date having been published for half a year in Jenkins already. Pulling the rug under admins now and taking away ~2/3 of the remaining time is not OK.

[the remark this answers is not in the document]
Comment (timja) [reply jep/236/README.adoc:237]: messaging in https://github.com/jenkinsci/jenkins/pull/6566

**Review by basil — APPROVED** [review]
Messaging could have been smoother, but at this point the important thing is to get it done.

**Pull request merged**